client/src/features/map-page/map-page.container.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { GoogleMapContainer } from '../google-map/google-map.container';
+
+export class MapPageContainer extends Component {
```

Comment:
since this component is just rendering a div, you can use a functional component instead; class is ideal for smart components that manages state; also because we want consistency throughout our app :)